.gitlab-ci.yml

```diff
@@ -0,0 +1,32 @@
+image: python:2.7
+
+cache:
+  paths:
+    - assembl/static2/node_modules/
+
+test_frontend:
+  before_script:
+    - apt-get update -qq
+    - apt-get remove -y -qq cmdtest
```

Comment:
yeah, we need to remove `cmdtest` because it provides a `yarn` binary that conflicts with yarn...
See https://yarnpkg.com/en/docs/install#debian-stable